website/docs/components/tag/partials/specifications/anatomy.md

```diff
@@ -52,31 +35,22 @@ As a best practice, we recommend a vertical spacing of 12px and horiztonal spaci
 
 ## Leading vs. Trailing Icon
 
-![Example of a leading icon in the tag](/assets/components/tag/tag-leading_icon.png)
 
 The leading dismiss icon allows mouse users to clear multiple tags quicker, because they do not need to move their mouse, especially when tags have a more extended width.
 
-<section>
-  <Hds::Badge @text="Do" @color="success" @icon="check-circle-fill" @iconPosition="leading" />
-</section>
-
-![Example of a trailing icon in the tag](/assets/components/tag/tag-trailing_icon.png)
+!!! Do
 
-When the dismiss icon is trailing, the mouse user must adjust positioning to clear tags, which results in slower task completion.
+![Example of a leading icon in the tag](/assets/components/tag/tag-leading_icon.png)
 
-<section>
-  <Hds::Badge @text="Don't" @color="critical" @icon="x-circle-fill" @iconPosition="leading" />
-</section>
 
----
+When the dismiss icon is trailing, the mouse user must adjust positioning to clear tags, which results in slower task completion.
+!!!
 
-## Content
 
-When content within the tag is not user-generated, it should be concise and consistent with the category name.
+!!! Dont
```

Comment:
[question] Is there a reason that it's "Dont" instead of "Don't" ?

If we need a one-word-with-no-punctuation solution, should we use "Avoid" instead?

Reply:
Good question. I think it would be a matter of updating the label [here](https://github.com/hashicorp/design-system/blob/65f12a411603c6c817466a6545a410f8b5c6edd6/website/app/components/doc/do-dont/index.js#L11).

Reply:
ok here's a PR: https://github.com/hashicorp/design-system/pull/922